ci/cron/src/Main.hs

```diff
@@ -261,3 +262,13 @@ main = do
             putStrLn "Updating versions.json & hidden.json"
             update_s3 temp_dir gh_versions
         reset_cloudfront
+
+check_signatures :: IO ()
+check_signatures = do
+    putStrLn "FIXME"
+
+main :: IO ()
```

Comment:
Any reason not to use `optparse-applicative` here? IME it’s pretty cheap in terms of the extra effort it requires even for very simple things and it’s much more maintainable as things get more complex.

Reply:
I honestly don't expect this to get any more complex than this 2-way dispatch for a very long time, but I'll look at optparse next time I need to make any change there.